ql/src/test/results/clientpositive/llap/vector_ptf_part_simple.q.out

```diff
@@ -314,46 +386,46 @@ POSTHOOK: type: QUERY
+Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	6	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond antique salmon chartreuse burlywood	1602.59	7	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond antique burnished rose metallic	1173.15	8	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond aquamarine burnished black steel	1414.42	9	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond aquamarine pink moccasin thistle	NULL	10	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	11	1	1	1632.66	1632.66	11	12
+Manufacturer#1	almond antique burnished rose metallic	1173.15	12	1	1	1632.66	1632.66	11	12
+Manufacturer#5	almond antique blue firebrick mint	1789.69	6	1	1	1611.66	1789.69	6	6
 Manufacturer#5	almond antique medium spring khaki	1611.66	1	1	1	1611.66	1789.69	6	6
 Manufacturer#5	almond aquamarine dodger light gainsboro	1018.1	2	1	1	1611.66	1789.69	6	6
 Manufacturer#5	almond azure blanched chiffon midnight	1464.48	3	1	1	1611.66	1789.69	6	6
 Manufacturer#5	almond antique medium spring khaki	1611.66	4	1	1	1611.66	1789.69	6	6
 Manufacturer#5	almond antique sky peru orange	1788.73	5	1	1	1611.66	1789.69	6	6
-Manufacturer#5	almond antique blue firebrick mint	1789.69	6	1	1	1611.66	1789.69	6	6
-Manufacturer#1	almond aquamarine pink moccasin thistle	1632.66	1	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	2	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond aquamarine pink moccasin thistle	1632.66	3	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	4	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond aquamarine pink moccasin thistle	1632.66	5	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	6	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique salmon chartreuse burlywood	1602.59	7	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique burnished rose metallic	1173.15	8	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond aquamarine burnished black steel	1414.42	9	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond aquamarine pink moccasin thistle	NULL	10	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique chartreuse lavender yellow	1753.76	11	1	1	1632.66	1173.15	11	12
-Manufacturer#1	almond antique burnished rose metallic	1173.15	12	1	1	1632.66	1173.15	11	12
+Manufacturer#4	almond aquamarine floral ivory bisque	1206.26	5	1	1	NULL	1206.26	4	6
+Manufacturer#4	almond azure aquamarine papaya violet	1290.35	6	1	1	NULL	1206.26	4	6
+Manufacturer#4	almond antique gainsboro frosted violet	NULL	1	1	1	NULL	1206.26	4	6
+Manufacturer#4	almond aquamarine floral ivory bisque	NULL	2	1	1	NULL	1206.26	4	6
+Manufacturer#4	almond antique violet mint lemon	1375.42	3	1	1	NULL	1206.26	4	6
+Manufacturer#4	almond aquamarine yellow dodger mint	1844.92	4	1	1	NULL	1206.26	4	6
+Manufacturer#2	almond antique violet turquoise frosted	1800.7	3	1	1	900.66	1800.7	8	8
```

Comment:
The result for last value column has changed wrt to old value,  `2031.98` -> `1800.7`. Why is that the case? Is it correct?
It seems the change also happened for other rows below.

Reply:
Yes, I still do not understand why the results are changing. But last value column expression looks like last_value(p_retailprice) over(partition by p_mfgr) as lv, which is without an order by clause. Because of which we have now changed the order of processing the rows and the result is also changed. I spent some time understanding the spec and how other databases behave when we do not have an order by clause. The spec does not say anything and also each database behave in a different way in these circumstances. Do you think it is ok for me to update the keys for this q file?